static/bidder-info/lemmadigital.yaml

```diff
@@ -0,0 +1,14 @@
+endpoint: "https://sg.ads.lemmatechnologies.com/lemma/servad?pid={{.PublisherID}}&aid={{.AdUnit}}"
+maintainer:
+  email: support@lemmatechnologies.com
```

Comment:
@lm-ved, PBS team has send mail for maintainer email address verification. Requesting to reply on email to complete verification

Reply:
@onkarvhanumante, replied to email.